build.gradle

```diff
@@ -208,6 +208,7 @@ dependencies {
     implementation 'io.fabric8:kubernetes-client:4.13.0'
 
     testImplementation "org.junit.jupiter:junit-jupiter-api:${junitVersion}"
+    testImplementation "org.junit.jupiter:junit-jupiter-params:${junitVersion}"
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junitVersion}"
```

Comment:
We can replace all 3 dependencies with `org.junit.jupiter:junit-jupiter:${junitVersion}`. It is an aggregator for those dependencies